### PR TITLE
Customize styles

### PR DESCRIPTION
**Pull request by verarojman**
Need to solve ⚠️ How to use variable font weights to have super-bold headings

- [x] Navbar
- [x] Process nav
- [x] Footer
- [x] Header
- [x] Buttons (especially hollow buttons)

This is a follow-up on #3  and #1

### CODE REVIEW

File: app/packs/stylesheets/decidim/_decidim-settings.scss [hunk 1]
```diff
@@ -5,3 +5,4 @@
 @include foundation-button;
 @include foundation-card;
 @include foundation-tabs;
+// @include foundation-forms;
```
Comment (fblupi) [inline app/packs/stylesheets/decidim/_decidim-settings.scss]: This line is commented. To keep the code cleaner, we should delete it unless it has been added for one reason I don't know

File: app/packs/stylesheets/desfake/_variables.scss [hunk 2]
```diff
@@ -211,12 +213,54 @@ $button-responsive-expanded: false;
...
+$card-border: 1px solid $light-gray;
 $card-shadow: none;
 $card-border-radius: $global-radius;
 $card-padding: $global-padding;
 $card-margin-bottom: $global-margin;
 
+// // 20. Forms
+// // ---------
+
+// $fieldset-border: 1px solid $primary-color;
```
Comment (fblupi) [inline app/packs/stylesheets/desfake/_variables.scss:223]: Like the other comment, many commented lines give me the impression they are here from a discarded or incomplete work.

**Review by fblupi — COMMENTED** [review]
Delete commented lines. The rest of the work has been reviewed and tested. Looks nice! ✨

File: app/packs/stylesheets/desfake/_variables.scss (hunk 2 above)
Comment (verarojman) [reply app/packs/stylesheets/desfake/_variables.scss:223]: I added a comment to explain why I'm leaving those lines commented out :)

**Review by fblupi — APPROVED** [review]

**Comment by verarojman** [comment]
Thanks @fblupi !

**Pull request merged**